app/types/configuration/build.d.ts

```diff
@@ -136,8 +136,6 @@ interface QuasarStaticBuildConfiguration {
    * When providing an object, it represents webpack-bundle-analyzer config options.
    */
   analyze: boolean | BundleAnalyzerPlugin.Options;
-  /** Include vue runtime + compiler version, instead of default Vue runtime-only. */
-  vueCompiler: boolean;
```

Comment:
Why removing this?

Reply:
I removed it in a previous iteration where I forced the usage of the alias for all builds removing the conditional branch
Then I added the ternary condition to use the esm runtime dist when vueCompiler wasn't needed and forgot to re-add the type

Fixed